include/irr/asset/IAssetManager.h

```diff
@@ -182,6 +182,19 @@ namespace asset
 			Furthermore if an Asset Bundle hasn't been loaded to cache before, it becomes loaded into cache memory and the function returns it. Otherwise
 			it is simply returned.
 
+			Take a look on @param _hierarchyLevel
```

Comment:
Super weird description, doesn't actually say anything that could be understood by a newb.